pkg/counter/cache.go

```diff
@@ -102,6 +110,31 @@ func DecrementUserAccountCount(log logr.Logger, clusterName string) {
 	})
 }
 
+// IncrementUsersPerActivationCounter increments the counter of a given number of activations for a (returning) user.
+// When a user signs up for the 1st time, her `activations` number is `1`, on the second time, it's `2`, etc.
+func IncrementUsersPerActivationCounter(activations int) {
+	write(func() {
+		// increase the gauge with the given number of activations
+		cachedCounts.UsersPerActivationCounts[strconv.Itoa(activations)]++
+		metrics.UsersPerActivationGaugeVec.WithLabelValues(strconv.Itoa(activations)).Inc()
+		// and decrease the gauge with the previous number of activations (if applicable)
+		if activations > 1 {
+			cachedCounts.UsersPerActivationCounts[strconv.Itoa(activations-1)]--
+			metrics.UsersPerActivationGaugeVec.WithLabelValues(strconv.Itoa(activations - 1)).Dec()
+		}
+	})
+}
+
+// DecrementUsersPerActivationCounter decrements the counter of a given number of activations for a (returning) user.
+// Happens when a user is deleted
```

Comment:
```suggestion
// Happens when a user is deactivated or deleted
```

Reply:
thins func is not used anymore, I removed it in 0cb3a75